bm-persona/Common/MissingDataView.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  MissingDataView.swift
+//  bm-persona
+//
+//  Created by Oscar Bjorkman on 4/4/20.
+//  Copyright © 2020 RJ Pimentel. All rights reserved.
+//
+
+import UIKit
+
+class MissingDataView: UIView {
+    
+    private var missingLabel: UILabel!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        let label = UILabel()
+        label.textColor = Color.darkGrayText
+        label.textAlignment = .center
+        label.font = Font.medium(18)
+        label.text = "No items found"
```

Comment:
we should probably have a file to store all constant string messages like this one (for consistency in the future). not too relevant for this pr

Reply:
for example "Nothing found!" when search has no results or "Unknown" when there's no phone number